deploy/operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       containers:
         - name: gitops-operator
           # Replace this with the built image name
-          image: REPLACE_IMAGE
+          image: quay.io/cbanavik/gitops-operator:role
```

Comment:
I think this is automatically replaced, and so we likely don't want your test image.

Reply:
This is outdated as well. I shouldn't have pushed it in the first place :)